backend/wmg/pipeline/constants.py

```diff
@@ -35,3 +35,87 @@
 WMG_DATA_SCHEMA_VERSION = "v4"
 
 MAXIMUM_ADMISSIBLE_CENSUS_SCHEMA_MAJOR_VERSION = 2
+
+HIGH_LEVEL_TISSUES = [
```

Comment:
if this is the same list / order as this [one](https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/ontology-assets/tissue_general_list.json) used by census, you can fetch it via COG using [this](https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/api/python/src/cellxgene_ontology_guide/curated_ontology_term_lists.py) function, passing in `CuratedOntologyTermList.TISSUE_GENERAL` enum